timingApp/Db/evg.db

```diff
@@ -56,6 +56,34 @@ record(mbbiDirect, "$(P)$(R)RxEnbl-RB") {
   field(INP, "@timing.proto evg_ctrl_rxen_intr $(PORT)")
 }
 
+record(bo, "$(P)$(R)RxLockedLtcRst-Cmd"){
+  field(DTYP, "stream")
+  field(DESC, "Reset Rx locked Latches")
+  field(MASK, "1")
+  field(HIGH, "1")
+  field(ZNAM, "nothing")
+  field(ONAM, "reset")
+  field(OUT, "@timing.proto evg_ctrl_set($(P),$(R)) $(PORT)")
+}
+
+record(mbbiDirect, "$(P)$(R)RxLocked-Mon") {
+  field(ASG, "Reserved")
+  field(DESC, "Each bit shows Rx locked status")
+  field(NOBT, "8")
+  field(SCAN, "I/O Intr")
+  field(DTYP, "stream")
+  field(INP, "@timing.proto evg_ctrl_rxlocked_intr $(PORT)")
+}
+
+record(mbbiDirect, "$(P)$(R)RxLockedLtc-Mon") {
+  field(ASG, "Reserved")
+  field(DESC, "Each bit shows Rx locked latches status")
+  field(NOBT, "8")
+  field(SCAN, "I/O Intr")
+  field(DTYP, "stream")
+  field(INP, "@timing.proto evg_ctrl_rxlockedltc_intr $(PORT)")
+}
+
```

Comment:
I think I'd prefer to have these as a separate DB, and we can pass `fout` or `evg` as a parameter. What do you think?

Shouldn't it be possible to have the `_intr` functions be the same for both devices as well?

Reply:
It would be better, but I don't know how to call different `ctrl_set` functions using the same DB. I think `*_ctrl_*_intr` functions are identical for both evg and fanout.